includes/handler/class-inbox.php

```diff
@@ -37,14 +40,28 @@ public static function init() {
 	 * @param Activity|\WP_Error $activity The Activity object.
 	 * @param string             $context  The context of the request.
 	 */
-	public static function maybe_handle_inbox_request( $data, $user_ids, $type, $activity, $context = Inbox_Collection::CONTEXT_INBOX ) {
+	public static function schedule_inbox_request( $data, $user_ids, $type, $activity, $context = Inbox_Collection::CONTEXT_INBOX ) {
 		// Ignore shared inbox requests to prevent duplicate processing.
 		if ( Inbox_Collection::CONTEXT_SHARED_INBOX === $context ) {
 			return;
 		}
 
-		// Process inbox requests.
-		self::handle_inbox_requests( $data, $user_ids, $type, $activity, $context );
+		$activity_id = isset( $data['id'] ) ? $data['id'] : null;
```

Comment:
```suggestion
		$activity_id = $data['id'] ?? null;
```
Would this be a good use case for `object_to_uri()`?